crates/ironrdp-pdu/src/write_buf.rs

```diff
@@ -40,6 +40,10 @@ impl WriteBuf {
         self.inner
     }
 
+    pub fn slice(&self, from: usize, to: usize) -> &[u8] {
+        &self.inner[from..to]
+    }
+
```

Comment:
You could implement the trait `Index<Range<usize>>` instead if you want

Reply:
fb16114b545a7a09232af3b2c93211460c6b7ad2